app/assets/javascripts/components/applicants/applicant_documents.jsx

```diff
@@ -4,34 +4,81 @@ const FILE_INPUTS = {
     PICTURE : "image/*"
```

Comment:
Not your fault, but do you mind changing this struct to FileInputs? There should only be like 3 calls to this throughout the entire codebase